lisa/wa.py

```diff
@@ -545,3 +547,35 @@ def __init__(self, wa_output, trace_to_df, **kwargs):
     def _get_artifact_df(self, path):
         trace = Trace(path, **self._trace_kwargs)
         return self._trace_to_df(trace)
+
+class WAJankbenchCollector(WAArtifactCollectorBase):
+    """
+    WA collector for the jankbench frame timings.
+
+    Each job will return a single DF which contains one row for each rendered frame.
+    The columns reported are:
+        * total_duration - (float) time in milliseconds to complete the frame
+        * jank_frame     - (int)   1 for a missed deadline, 0 otherwise
+        * name           - (string)subtest name from Jankbench
+        * frame_id       - (int)   frame order, increasing
+
+    An example plotter can be found in the jupyter notebook working directory
+    at <ipynb>/wltests/WAOutput-JankbenchDemo.ipynb
+    """
+
+    NAME = 'jankbench'
+    _ARTIFACT_NAME = "jankbench-results"
+    _ARTIFACT_METRIC_SUFFIX_UNIT = defaultdict(
+        # Default is an empty string
+        str,
+        {
+            'total_duration': 'ms',
+        },
+    )
+
+    def _get_artifact_df(self, path):
+        con = sqlite3.connect(path)
+        raw_df = pd.read_sql_query("SELECT * from ui_results", con)
```

Comment:
We might want to select the columns we are actually interested in, since we already have a hardcoded list of them anyway. This should make loading faster and less memory hungry

Reply:
Yep, this is left over from when I wasn't so sure what I wanted to show. Fixed.